README.md

```diff
@@ -23,3 +23,9 @@ The project can be build by using `make`. Along with the usual `make`, `make cle
 Tests are located in `demo/` and can be built in two flavors.
 * `make test` creates a test binary that is intended to be run against `mtest`. `mtest` can be built with `make mtest` and test execution is done like `./mtest/mtest | ./test`. `mtest` is creating test vectors using an alternative MPI library and `test` is consuming these vectors to verify correct behavior of ltm
 * `make test_standalone` creates a stand-alone test binary that executes several test routines.
+
+[Valgrind](http://valgrind.org/) does not support `long double` but does not reset the limits in `float.h` accordingly. Please build `demo.c` with the additional compiler flag `-DLTM_MEMCHECK_VALGRIND`, e.g.:
+
+`CFLAGS=" -DLTM_MEMCHECK_VALGRIND "  make test_standalone`
```

Comment:
I suggest the has_include check, see other comment.

Reply:
It is not in the C-standard (c-11, too stingy for c18) but in >=C++17. It is supported in clang, GCC (since v5), and  MSVC (see e.g.: https://en.cppreference.com/w/cpp/compiler_support). My native GCC is too old (4.9) and doesn't know it, the version of GCC-5 here is "5.5.0 20171010" and does know it. 

Mmh&hellip;no, I wouldn't use it. At least not for now and not for a single special case that is even for testing only.